pkgs/top-level/release.nix

```diff
@@ -202,6 +202,7 @@ let
       pkgsLLVM.stdenv = [ "x86_64-linux" "aarch64-linux" ];
       pkgsMusl.stdenv = [ "x86_64-linux" "aarch64-linux" ];
       pkgsStatic.stdenv = [ "x86_64-linux" "aarch64-linux" ];
+      pkgsStatic.rc = [ "x86_64-linux" ];
```

Comment:
I am not really sure how this file works and what should be added here or not. @jtojnar @jonringer can you answer that or know someone that knows that?